src/Jsonyte/Serialization/Metadata/JsonTypeInfo.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System;
+﻿using Jsonyte.Serialization.Attributes;
```

Comment:
Nit: System usings should appear before others. Move Jsonyte using to the bottom.